src/recipe.c

```diff
@@ -445,7 +446,14 @@ recipe_parse (xmlDoc *doc, SoupURI *recipe_uri, GError **error, gchar **cfg_file
         g_free (tmp_str);
     }
     result->recipe_uri = recipe_uri;
-    result->base_path = TASK_LOCATION;
+
+    // Gather the location in which to install tasks
+    result->base_path = get_install_dir(INSTALL_CONFIG_FILE, &tmp_error);
+    if (tmp_error != NULL) {
+        g_warning("* Fail getting task install path using default. Error: %s",
+                  tmp_error->message);
```

Comment:
`tmp_error` is used later, so it should be cleared after the `g_warning` call.

Reply:
Agreed. I just stumbled across this.